### PR TITLE
feat: run loadgen for a few seconds during installation

### PR DESCRIPTION
**Pull request by nicoleczhu**
fix: #601 
Run load generator for ~10-15s during installation. 
Adds ~5s to previous install time. 

**Load specs:** 
user_count=50
spawn_rate=10
At the end of 5 seconds we delete users that have already been created. 

**Note:** 
There's a race condition where `curl http://loadgen_ip/stop` to stop loadgen returns a successful http response (and UI indicates the swarm has terminated..) 

BUT this only terminates users that have already been spawned, not the users that were originally scheduled to spawn! [Bug filed with Locust here](https://github.com/locustio/locust/issues/1661). 

For us this might be nice as sandbox users will continue to get traces/metrics from any undeleted remaining users from Locust.

**Context:** 
We wanted to provide initial traffic telemetry for users to view && still provide Locust loadgen UI. 
Implemented the workaround suggested by this [Locust issue](https://github.com/locustio/locust/issues/831)

**Testing:** 
In Span Viewer, use filter `/http/user_agent:python-requests`

### CODE REVIEW

File: terraform/install.sh [hunk 1]
```diff
@@ -283,16 +284,26 @@ loadGen() {
   gcloud container clusters get-credentials loadgenerator --zone "$LOADGEN_ZONE"
   kubectx loadgenerator=.
 
-  # find the IP of the load generator web interface
+  # Kicks of initial load generation via a POST request to Locust web interface
+  # This is because Locust currently doesn't support CLI and UI at the same time
   TRIES=0
-  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
+#  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
```
Comment (Daniel-Sanche) [inline terraform/install.sh:290]: should this comment stick around?

File: terraform/install.sh [hunk 2]
```diff
@@ -283,16 +284,26 @@ loadGen() {
...
+  while [[ $(curl -XPOST -d "user_count=100&spawn_rate=10" http://$loadgen_ip/swarm -o /dev/null -w "%{http_code}" --max-time 1) -ne 200  && \
```
Comment (Daniel-Sanche) [inline terraform/install.sh:291]: is /swarm idempotent? Can we keep trying to hit it multiple times and always expect a 200 response?

File: terraform/install.sh [hunk 3]
```diff
@@ -283,16 +284,26 @@ loadGen() {
...
     TRIES=$((TRIES + 1))
   done
+
+  # Lets load generator run for a few seconds
+  sleep 5
+
+  # Ends initial load generation
+  if [[ $(curl http://$loadgen_ip/stop -o /dev/null -w "%{http_code}") -ne 200 ]]; then
+    log "Failed to stop initial load generation"
+  fi
```
Comment (Daniel-Sanche) [inline terraform/install.sh:306]: Are you sure we want to shut down the load generation? Maybe we should keep it running so users continuously see things come in

File: terraform/install.sh [hunk 4]
```diff
@@ -283,16 +284,26 @@ loadGen() {
   gcloud container clusters get-credentials loadgenerator --zone "$LOADGEN_ZONE"
   kubectx loadgenerator=.
 
-  # find the IP of the load generator web interface
+  # Kicks of initial load generation via a POST request to Locust web interface
```
Comment (simonz130) [inline terraform/install.sh:287]: nit: Kicks off

**Review by simonz130 — APPROVED** [review]

File: terraform/install.sh (hunk 3 above)
Comment (nicoleczhu) [reply terraform/install.sh:306]: I'm a fan of this suggestion. This actually solves the Locust bug issue.

File: terraform/install.sh (hunk 2 above)
Comment (nicoleczhu) [reply terraform/install.sh:291]: It is indempotent

File: terraform/install.sh (hunk 4 above)
Comment (nicoleczhu) [reply terraform/install.sh:287]: thanks!

File: terraform/install.sh (hunk 1 above)
Comment (nicoleczhu) [reply terraform/install.sh:290]: removed. thanks!

File: terraform/install.sh (hunk 3 above)
Comment (nicoleczhu) [reply terraform/install.sh:306]: Hm another Locust bug. If we start the swarm with a URL post request, users don't get the `STOP` button in the UI. 😬😬😬

The fundamental issue here is the Locust page is all jquery. I'll reduce the swarm size and increase the swarm run time by a little bit.

**Comment by nicoleczhu** [comment]
Ok this is mergeable

**Pull request merged**